src/cgroup.rs

```diff
@@ -59,19 +59,26 @@ impl Default for Cgroup {
 }
 
 impl Cgroup {
+    pub fn v2(&self) -> bool {
+        self.hier.v2()
+    }
+
     /// Create this control group.
     fn create(&self) {
         if self.hier.v2() {
-            let _ret = create_v2_cgroup(self.hier.root(), &self.path);
+            let _ret = create_v2_cgroup(self.hier.root(), &self.path, None);
         } else {
             for subsystem in &self.subsystems {
                 subsystem.to_controller().create();
             }
         }
     }
 
-    pub fn v2(&self) -> bool {
-        self.hier.v2()
+    /// Create this control group with specified controllers.
+    ///
+    /// Note that this method is only meaningful for cgroup v2.
```

Comment:
Why only v2 supported this? May be v1 also need this feature?

Reply:
I think that this feature is required in v1 cgroup, too.  In some scenes,  we just want to add the process to those specified subsystems.

Reply:
Does this mean that only specified controllers support threaded mode, so we should create with specified controller?

Reply:
Thank you for your review @lifupan @gaohuatao-1 @YushuoEdge. I added the method `load_with_specified_controllers()` for specified section controllers in cgroup v1.